Gemfile.lock

```diff
@@ -339,8 +342,6 @@ GEM
     globalid (0.5.2)
       activesupport (>= 5.0)
     google-protobuf (3.19.1)
-    google-protobuf (3.19.1-x86_64-darwin)
-    google-protobuf (3.19.1-x86_64-linux)
```

Comment:
Looks like there was some Bundler change where this works without specifying architecture and OS. Just checked this both on Intel and Apple M1 after `rvm gemset empty` and it works. 🤯